amps/ags/rm-community/rm-community-repo/source/java/org/alfresco/module/org_alfresco_module_rm/capability/policy/CreatePolicy.java

```diff
@@ -42,10 +43,23 @@ public int evaluate(
     {
         NodeRef linkee = null;
         QName assocType = null;
+        QName recordType = null;
 
         // get the destination node
         NodeRef destination = getTestNode(invocation, params, cad.getParameters().get(0), cad.isParent());
 
+        //get the recordType
+        for (Object qname : invocation.getArguments()) {
+            if (qname!= null && qname.equals(RecordsManagementModel.TYPE_RECORD_FOLDER))
+            {
+                recordType = (QName) qname;
+            }
+            else if (qname!= null && qname.equals(RecordsManagementModel.TYPE_RECORD_CATEGORY))
+            {
+                recordType = (QName) qname;
+            }
```

Comment:
Can we combine both if else conditions in a single if statement as both blocks seems to perform similar function

Reply:
yes, I have done the changes as per the comment.